ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -69,6 +73,24 @@
             ctx)
            {:status 200}))))
 
+(defn- add-cmr-ingest-to-context
```

Comment:
Similarly I'd change the name of this function.

Reply:
Done.